apps/src/applab/DesignModeBox.jsx

```diff
@@ -13,6 +13,7 @@ module.exports = React.createClass({
     onDelete: React.PropTypes.func.isRequired,
     onInsertEvent: React.PropTypes.func.isRequired,
     isToolboxVisible: React.PropTypes.bool.isRequired,
+    isApplabRunning: React.PropTypes.bool.isRequired
```

Comment:
naming nit: I would prefer to call this `isDimmed`, so that we are telling the component what to do rather than telling it the state of the world and then letting it decide what to do based on that.  From a practical perspective, we may want to dim it for other reasons besides `isApplabRunning` in the future. Feel free to change it or leave it depending on how strongly you agree or disagree :)